sw/ground_segment/python/fuelcell/fuel_cell_viewer.py

```diff
@@ -0,0 +1,548 @@
+        self.StatusBox(dc, dx, dy, 1, 0, self.fuelcell.get_tank(), self.fuelcell.get_tank_perc(), self.fuelcell.get_tank_color())
+        self.StatusBox(dc, dx, dy, 2, 0, self.fuelcell.get_battery(), self.fuelcell.get_battery_perc(), self.fuelcell.get_battery_color())
+        self.StatusBox(dc, dx, dy, 3, 0, self.fuelcell.get_status(), self.fuelcell.get_status_perc(), self.fuelcell.get_status_color())
+        self.StatusBox(dc, dx, dy, 4, 0, self.fuelcell.get_error(), self.fuelcell.get_error_perc(), self.fuelcell.get_error_color())
+
+        # Warnings
+        self.stat = int(0.14*w)
+        dc.SetBrush(wx.Brush(wx.Colour(70,70,40))) 
+        dc.DrawRectangle(int(0.36*w), int(0.63*h),int(0.28*w), int(0.35*h))
+        dx = int(0.36*w)
+        dy = int(0.63*h)
+        for i in range(0,8,1):
+            self.StatusBox(dc, dx, dy,i, 0, self.fuelcell.get_error_nr(i*2), 1, self.fuelcell.get_error_nr_color(i*2))
+            self.StatusBox(dc, int(0.5*w), dy,i, 0, self.fuelcell.get_error_nr(i*2+1), 1, self.fuelcell.get_error_nr_color(i*2+1))
+
+        # Motors
+        self.stat = int(0.10*w)
+
+        dc.SetBrush(wx.Brush(wx.Colour(200,200,100))) 
+        w1 = 0.20
+        w2 = 0.60
+        dw = 0.11
+        mw = 0.1
+        mm = [(0.03,w1), (0.03+dw,w1), (0.03+2*dw,w1), (0.97-mw-2*dw,w1), (0.97-mw-dw,w1), (0.97-mw,w1), (0.03,w1+0.17), (0.03+dw,w1+0.17), (0.97-mw-dw,w1+0.17), (0.97-mw,w1+0.17),
+              (0.03,w2), (0.03+dw,w2), (0.03+2*dw,w2), (0.97-mw-2*dw,w2), (0.97-mw-dw,w2), (0.97-mw,w2), (0.03,w2+0.17), (0.03+dw,w2+0.17), (0.97-mw-dw,w2+0.17), (0.97-mw,w2+0.17)]
+        for m in mm:
+            dc.DrawRectangle(int(m[0]*w), int(m[1]*h),int(mw*w), int(0.15*h))
+
+        for m in self.motors.mot:
+            mo_co = mm[m.id]
+            #print(m.id, mo_co)
+            dx = int(mo_co[0]*w)
+            dy = int(mo_co[1]*h)
+            self.StatusBox(dc, dx, dy, 0, 0, m.get_volt(), m.get_volt_perc(), 1)
+            self.StatusBox(dc, dx, dy, 1, 0, m.get_current(), m.get_current_perc(), 1)
+            self.StatusBox(dc, dx, dy, 2, 0, m.get_rpm(), m.get_rpm_perc(), m.get_rpm_color)
+            self.StatusBox(dc, dx, dy, 3, 0, m.get_temp(), m.get_temp_perc(), m.get_temp_color())
+
+
+
+    def __init__(self):
+
+        self.w = WIDTH
+        self.h = WIDTH + BARH
+
+        self.cfg = wx.Config('fuel_cell_conf')
+        if self.cfg.Exists('width'):
+            self.w = int(self.cfg.Read('width'))
+            self.h = int(self.cfg.Read('height'))
+
+        wx.Frame.__init__(self, id=-1, parent=None, name=u'FuelCellFrame',
+                          size=wx.Size(self.w, self.h), title=u'Fuel Cell Monitoring')
+
+        if self.cfg.Exists('left'):
+            self.x = int(self.cfg.Read('left'))
+            self.y = int(self.cfg.Read('top'))
+            self.SetPosition(wx.Point(self.x,self.y), wx.SIZE_USE_EXISTING)
```

Comment:
this method only take one argument, the second one should be removed.

Reply:
Removed in all python tools that have this